roles/openshift_health_checker/openshift_checks/fluentd_config.py

```diff
@@ -0,0 +1,188 @@
+        """Read and return the value of the 'USE_JOURNAL' environment variable on a fluentd pod."""
+        running_pods = self.running_fluentd_pods(task_vars)
+        pod_containers = running_pods[0]["spec"]["containers"]
+
+        if not pod_containers:
+            msg = ('There are no running containers on selected Fluentd pod "{}".\n'
+                   'Unable to calculate expected logging driver.').format(running_pods[0]["metadata"]["name"])
+            raise OpenShiftCheckException(msg)
+
+        pod_env = pod_containers[0]["env"]
+        if not pod_env:
+            msg = ('There are no environment variables set on the Fluentd container "{}".\n'
+                   'Unable to calculate expected logging driver.').format(pod_containers[0]["name"])
+            raise OpenShiftCheckException(msg)
+
+        for env in pod_env:
+            if env["name"] == "USE_JOURNAL":
+                return env["value"] != "false"
+
+        return False
+
+    def running_fluentd_pods(self, task_vars):
+        """Return a list of running fluentd pods."""
+        fluentd_pods, error = self.get_pods_for_component(
+            self.execute_module,
+            self.logging_namespace,
+            "fluentd",
+            task_vars,
+        )
+        if error:
+            msg = 'Unable to retrieve any pods for the "fluentd" logging component: {}'.format(str(error))
+            raise OpenShiftCheckException(msg)
+
+        running_fluentd_pods = [pod for pod in fluentd_pods if pod['status']['phase'] == 'Running']
+        if not len(running_fluentd_pods):
+            msg = ('No Fluentd pods were found to be in the "Running" state. '
+                   'At least one Fluentd pod is required in order to perform this check.')
+
+            raise OpenShiftCheckException(msg)
+
+        return running_fluentd_pods
+
+    def get_pods_for_component(self, execute_module, namespace, logging_component, task_vars):
+        """Get all pods for a given component. Returns: list of pods for component, error string."""
+        try:
+            pod_output = self.exec_oc(
+                execute_module,
+                namespace,
+                "get pods -l component={} -o json".format(logging_component),
+                [],
+                task_vars
+            )
+        except OpenShiftCheckException as err:
+            if "false" not in str(err):
+                raise err
+
+        try:
+            pods = json.loads(pod_output)
+            if not pods or not pods.get('items'):
+                raise ValueError()
+        except ValueError:
+            # successful run but non-parsing data generally means there were no pods in the namespace
+            return None, 'There are no pods in the {} namespace. Is logging deployed?'.format(namespace)
+
+        return pods['items'], None
+
+    @staticmethod
+    def exec_oc(execute_module=None, namespace="logging", cmd_str="", extra_args=None, task_vars=None):
```

Comment:
Save for white space differences in the docstring and a bug that was fixed in https://github.com/openshift/openshift-ansible/commit/5f51278dee925c47055aacbb8ffac5e351ea7ee2, this is identical to [LoggingCheck.exec_oc](https://github.com/openshift/openshift-ansible/blob/a41fc449c3e56646365701e5b7a40b29eb9e3b67/roles/openshift_health_checker/openshift_checks/logging/logging.py#L67).

Let's not duplicate this method please, nor reintroduce the bug.